docs/administrator.md

```diff
@@ -649,26 +618,108 @@ A secret can be pre-provisioned in different ways:
 * Automatically provisioned via a custom K8s controller like
   [kube-aws-iam-controller](https://github.com/mikkeloscar/kube-aws-iam-controller)
 
-## Google Cloud Platform setup
+## WAL archiving and basebackups
 
-To configure the operator on GCP there are some prerequisites that are needed:
+Spilo is shipped with [WAL-E](https://github.com/wal-e/wal-e) and its successor
+[WAL-G](https://github.com/wal-g/wal-g) to perform WAL archiving. By default,
+WAL-E is used because it is more battle-tested. Additionally to the continuous
+backup stream a [basebackup](https://www.postgresql.org/docs/13/app-pgbasebackup.html)
+is initialized every night and 1am UTC.
+
+These are the pre-configured settings in the docker image:
+
+```bash
+archive_command:  `envdir "{WALE_ENV_DIR}" {WALE_BINARY} wal-push "%p"`
+restore_command:  `envdir "{{WALE_ENV_DIR}}" /scripts/restore_command.sh "%f" "%p"`
```

Comment:
Logs from `archive_command` and `restore_command` are written to the `$HOME/pgdata/pgroot/pg_log/postgres-?.log`